src/ophys_etl/modules/neuropil_correction/utils.py

```diff
@@ -0,0 +1,404 @@
+import logging
+from pathlib import Path
+from typing import List, Tuple, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.sparse as sparse
+from scipy.linalg import solve_banded
+
+matplotlib.use("agg")
+
+
+def debug_plot(
+    file_name: Union[str, Path],
+    roi_trace: np.ndarray,
+    neuropil_trace: np.ndarray,
+    corrected_trace: np.ndarray,
+    r: float,
+    r_vals: np.ndarray = None,
+    err_vals: np.ndarray = None,
+) -> None:
+    """Create debug plot for neuropil correction"""
```

Comment:
There's a few functions that have full docstrings and others that don't. Could you take a crack at adding the full docstring?